src/sentry/mail/adapter.py

```diff
@@ -115,21 +117,23 @@ def get_sendable_user_objects(project):
             organization_id=project.organization_id,
             actor_type=ActorType.USER,
         )
-        return recipients.get(ExternalProviders.EMAIL)
+        result = recipients.get(ExternalProviders.EMAIL)
+        assert result is not None
+        return result
```

Comment:
a call to `get_sendable_user_objects` assumes it is always a set, so setting the return type to `set[Actor] | None` makes mypy yell

chose to fail loudly as it seems there should always be a result